sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/json/JsonFileFormat.scala

```diff
@@ -36,29 +31,31 @@ import org.apache.spark.sql.sources._
 import org.apache.spark.sql.types.StructType
 import org.apache.spark.util.SerializableConfiguration
 
+object JsonFileFormat {
+  def parseJsonOptions(sparkSession: SparkSession, options: Map[String, String]): JSONOptions = {
```

Comment:
I think I disagree with passing whole `SparkSession` because apparently we only need `SQLConf` or the value of `spark.sql.columnNameOfCorruptRecord`.